src/api/klamm/klamm.service.ts

```diff
@@ -73,6 +73,15 @@ export class KlammService {
     }
   }
 
+  async getKlammBRERules(): Promise<string[]> {
+    try {
+      const { data } = await this.axiosKlammInstance.get(`${process.env.KLAMM_API_URL}/api/brerules`);
+      return data;
+    } catch (err) {
+      throw new HttpException('Error fetching from Klamm', HttpStatus.INTERNAL_SERVER_ERROR);
+    }
+  }
```

Comment:
`_getAllKlammFields` already does this. Maybe remove that method and have the code call this one instead?

Reply:
Feel silly for missing this! Thank you, will update.

Reply:
Removed the duplication and updated BRM App PR https://github.com/bcgov/brm-backend/pull/58 to match.